packages/tracking/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "1.0.2",
+  "version": "2.0.0",
```

Comment:
What do you think?

Reply:
Probably indeed a v2 since I remember peters mantra: Updated peerDependencies === Updated Major Version :-)

Reply:
This PR is **adding** support for react 18, so it's not a breaking change -> therefore it's 1.1.0

Reply:
Wrote a small section on the wiki about it 

https://wiki.nebenan.de/display/AWSM/1st+Party+Packages#id-1stPartyPackages-Situation2.1:Addingsupportformoreversions